modules/edu.gemini.seqexec.engine/src/test/scala/edu/gemini/seqexec/engine/StepSpec.scala

```diff
@@ -153,18 +157,18 @@ class StepSpec extends FlatSpec {
 
   }
 
-  val result = Result.OK(Unit)
+  val result = Result.OK(Result.Observed("dummyId"))
   val failure = Result.Error(Unit)
   val action: Action = Task(result)
   val config: StepConfig = Map()
-  val stepz0: Step.Zipper   = Step.Zipper(0, config, Nil, Execution.empty, Nil, (Execution.empty, Nil))
-  val stepza0: Step.Zipper  = Step.Zipper(1, config, List(List(action)), Execution.empty, Nil, (Execution.empty, List(List(action))))
-  val stepza1: Step.Zipper  = Step.Zipper(1, config, List(List(action)), Execution(List(result.right)), Nil, (Execution(List(action.left)), List(List(action))))
-  val stepzr0: Step.Zipper  = Step.Zipper(1, config, Nil, Execution.empty, List(List(result)), (Execution(List(action.left)), Nil))
-  val stepzr1: Step.Zipper  = Step.Zipper(1, config, Nil, Execution(List(result.right, result.right)), Nil, (Execution(List(action.left, action.left)), Nil))
-  val stepzr2: Step.Zipper  = Step.Zipper(1, config, Nil, Execution(List(result.right, result.right)), List(List(result)), (Execution(List(action.left)), List(List(action, action))))
-  val stepzar0: Step.Zipper = Step.Zipper(1, config, Nil, Execution(List(result.right, action.left)), Nil, (Execution(List(action.left, action.left)), Nil))
-  val stepzar1: Step.Zipper = Step.Zipper(1, config, List(List(action)), Execution(List(result.right, result.right)), List(List(result)), (Execution(List(action.left)), List(List(action, action), List(action))))
+  val stepz0: Step.Zipper   = Step.Zipper(0, None, config, Nil, Execution.empty, Nil, (Execution.empty, Nil))
+  val stepza0: Step.Zipper  = Step.Zipper(1, None, config, List(List(action)), Execution.empty, Nil, (Execution.empty, List(List(action))))
+  val stepza1: Step.Zipper  = Step.Zipper(1, None, config, List(List(action)), Execution(List(result.right)), Nil, (Execution(List(action.left)), List(List(action))))
+  val stepzr0: Step.Zipper  = Step.Zipper(1, None, config, Nil, Execution.empty, List(List(result)), (Execution(List(action.left)), Nil))
+  val stepzr1: Step.Zipper  = Step.Zipper(1, None, config, Nil, Execution(List(result.right, result.right)), Nil, (Execution(List(action.left, action.left)), Nil))
+  val stepzr2: Step.Zipper  = Step.Zipper(1, None, config, Nil, Execution(List(result.right, result.right)), List(List(result)), (Execution(List(action.left)), List(List(action, action))))
+  val stepzar0: Step.Zipper = Step.Zipper(1, None, config, Nil, Execution(List(result.right, action.left)), Nil, (Execution(List(action.left, action.left)), Nil))
+  val stepzar1: Step.Zipper = Step.Zipper(1, None, config, List(List(action)), Execution(List(result.right, result.right)), List(List(result)), (Execution(List(action.left)), List(List(action, action), List(action))))
```

Comment:
I modified this a bit in my PR to make it easier to adapt to changes to Step.Zipper